Rakefile

```diff
@@ -7,6 +7,7 @@ require 'rubygems'
 require 'rake/testtask'
 require 'yard'
 require "#{File.dirname(__FILE__)}/lib/tasks/all.rb"
+require_relative 'rakefile_helpers'
```

Comment:
All edits that are moving methods from rake tasks follow the same pattern:
- Remove methods from inside a task
- Add methods to a new Class defined in a new file `#{rakefile_name}_helpers.rb`
- Require the new helper file in .rake file and call the methods using newly created Class